src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -426,9 +426,17 @@ CameraTrigger::shoot_once()
 	}
 }
 
-void
+bool
 CameraTrigger::start()
 {
+	if (_camera_interface == nullptr) {
+		if (camera_trigger::g_camera_trigger != nullptr) {
+			delete (camera_trigger::g_camera_trigger);
```

Comment:
You need to set `g_camera_trigger` to null.

Reply:
@bkueng Done!